packages/salesforcedx-vscode-core/test/jest/commands/refreshSObjectsExecutor.test.ts

```diff
@@ -46,6 +46,17 @@ describe('RefreshSObjectsExecutor', () => {
     expect(channelServiceSpy).not.toHaveBeenCalled();
   });
 
+  it('should fire the command completion event once the command is finished successfully', async () => {
+    const fireSpy = jest.spyOn(
+      RefreshSObjectsExecutor.refreshSObjectsCommandCompletionEventEmitter,
+      'fire'
+    );
+
+    await doExecute(SObjectRefreshSource.Startup, SObjectCategory.STANDARD);
+
+    expect(fireSpy).toHaveBeenCalled();
```

Comment:
```suggestion
        expect(fireSpy).toHaveBeenCalledWith({
      exitCode: LocalCommandExecution.SUCCESS_CODE
    });
```

Reply:
This is a nit. also remember to add 
```suggestion
    import {
  LocalCommandExecution
} from '@salesforce/salesforcedx-utils-vscode';
```